### PR TITLE
Task/api 1244 fast and dangerous

### PR DESCRIPTION
**Pull request by ilaflamme-va**
- New Danger Zone options when deploying
- Manual approval required

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -80,7 +80,14 @@ test -n "${#DU_LOAD_BALANCER_RULES[@]}"
 export LOAD_BALANCER_RULES="$WORKSPACE/lb-rules.conf"
 declare -p DU_LOAD_BALANCER_RULES > $LOAD_BALANCER_RULES
 
-
+#
+# If we're in the DANGER ZONE, DU_VERSION might have been overwritten, lets check.
+#
+test -n "$FAST_AND_DANGEROUS_DU_VERSION"
```
Comment (bschofield-va) [inline build.sh]: I think the danger version should only be used if in danger mode

File: build.sh [hunk 2]
```diff
@@ -168,28 +185,34 @@ do
...
-  echo "Applying kubernetes configuration"
-  cluster-fox kubectl $AVAILABILITY_ZONE -- apply -v 5 -f $DU_DIR/deployment.yaml
-  attach-deployment-unit-to-lb green
-  wait-for-lb green
-
-  if ! execute-tests regression-test $GREEN_LOAD_BALANCER $AVAILABILITY_ZONE $DU_DIR $LOG_DIR
+  #
+  # If we are in the danger zone, skip all non-essential deployment steps.
+  #
+  if [ "$FAST_AND_DANGEROUS_BUILD" == false ]
```
Comment (bschofield-va) [inline build.sh]: This logic isn't quite right ...

Comment (ilaflamme-va) [reply build.sh]: Addressed.

File: build.sh (hunk 1 above)
Comment (ilaflamme-va) [reply build.sh]: Addressed.

**Pull request merged**